pandas/io/msgpack/_packer.pyx

```diff
@@ -1,15 +1,17 @@
 # coding: utf-8
 # cython: embedsignature=True
 
-from cpython cimport (
-    PyFloat_Check, PyLong_Check, PyInt_Check,
-    PyDict_CheckExact, PyDict_Check,
-    PyTuple_Check, PyList_Check,
-    PyCallable_Check,
-    PyUnicode_Check, PyBytes_Check,
-    PyBytes_AsString,
-    PyBytes_FromStringAndSize,
-    PyUnicode_AsEncodedString)
+from cpython.bytes cimport (PyBytes_Check, PyBytes_AsString,
+                            PyBytes_FromStringAndSize)
+from cpython.dict cimport PyDict_Check, PyDict_CheckExact
+from cpython.float cimport PyFloat_Check
+from cpython.int cimport PyInt_Check
+from cpython.list cimport PyList_Check
+from cpython.long cimport PyLong_Check
+from cpython.object cimport PyCallable_Check
+from cpython.tuple cimport PyTuple_Check
+from cpython.unicode cimport PyUnicode_Check, PyUnicode_AsEncodedString
```

Comment:
doesnt need to be for this PR, but most/all of the PyFoo_Check calls can be replaced with their pure-python equivalents; cython will now automatically make the optimization